server/user.go

```diff
@@ -43,15 +43,35 @@ func (c *Connection) JiraAccountID() types.ID {
 }
 
 type ConnectionSettings struct {
-	Notifications bool `json:"notifications"`
+	Notifications                *bool `json:"notifications"`
```

Comment:
why pointer values? what is the difference of a `nil` and a `false`?

Reply:
@levb this was a suggestion from @mickmister https://github.com/mattermost/mattermost-plugin-jira/pull/705#discussion_r631444153 
it seems to keep a track where the user have explicitly turned off the notification.

Reply:
@levb We're using pointers for backwards compatibility, so we can fall back on the user's current settings if these are `nil`